### ISSUE TITLE
Feature Request: Complete chat suite

### ISSUE THREAD

**Issue by viet** [minecraftly/minecraftly#33]
Public messaging modes:
- Global is only global per instance (default)
  - Why? This is to prevent the chat box gets filled up quickly when there are thousands of players online. Basically people can chat with people who are most relevant to them.
- Per world

How to switch between chat modes? Use /chat command

Private messaging
- /msg
  - Alias: /m, /tell, /message
- /reply
  - Alias: /r

**Comment by viet** [comment]
Yeah. After a while, it's important to have per bukkit server chat as default, which includes all players from multiple worlds.

Because we separate worlds, we gotta compensate for that separation via global chat right away, without any learning curve needed (e.g: /y or !)

**Comment by SilverCory** [comment]
I couldn't agree more. One thought I've always had is per player formatting. By default all messages are starred out, but you can turn it off..

**Comment by viet** [comment]
What do you mean by "per player formatting" and "all messages are starred out"

**Comment by SilverCory** [comment]
Sorry, per world filtering.
And instead of fuck it will be ****

**Comment by viet** [comment]
How is per world filtering different from a generic chat filtering plugin (eg. PwnFilter, BungeeChatFilter)?
